.gitignore

```diff
@@ -0,0 +1,9 @@
+PythonTool/visitlog.py
+Data/*.vtk
+Data/*.stl
+Data/*.h5m
+Data/meshtal
+Sessions
+Images
+PythonTool/*.pyc
+BashTool/DataFileInfo/*.txt
```

Comment:
This wasn't quite what I was referring to. There were some odd characters at the end of the file that github was rendering. They seem to be gone now.